src/main/java/us/kilroyrobotics/subsystems/CoralIntakeMotor.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package us.kilroyrobotics.subsystems;
+
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.epilogue.Logged;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import us.kilroyrobotics.Constants.CoralMechanismConstants;
+
+public class CoralIntakeMotor extends SubsystemBase {
+    private SparkMax wheelMotor;
+    private DigitalInput coralSensor = new DigitalInput(0);
+
+    /** Creates a new CoralIntakeMotor. */
+    public CoralIntakeMotor() {
+        this.wheelMotor = new SparkMax(CoralMechanismConstants.kWheelMotorId, MotorType.kBrushless);
```

Comment:
No configuration needed? No current limits need set?

Even if you don't set any config values, you should ALWAYS apply a blank config with resetSafe and Persist so you know that whatever sparkmax you connect will perform the same and have the same configuration